source/agora/node/Ledger.d

```diff
@@ -878,7 +878,7 @@ unittest
     // and one transaction has eight Outputs with a value of 40,000 values.
     void splitGenesis ()
     {
-        splited_txex = splitGenesisTransaction(getGenKeyPairs(), splited_keys);
+        splited_txex = splitGenesisTransaction(splited_keys);
```

Comment:
Can we also fix this typo? it's `splitted`, everywhere.

Reply:
Actually it should be `split`, not splitted maybe? hmm

Reply:
I plan to nuke those functions, but can fix the typo now if you'd like